src/styles/main.scss

```diff
@@ -1,3 +1 @@
-body {
-  margin: 0;
-}
+@import "header", "navigation", "container", "card", "reset";
```

Comment:
Improper import of styles. In this case, we do not know where the files with the styles we need are located.
you need to register here
`@import from (path to css file)`

reset file import always first because the styles are always read by the browser from top to bottom and if you connect them last, they will reset all the styles that were specified in other files